ansible/playbooks/jenkins/worker/create.yml

```diff
@@ -67,7 +67,7 @@
   - "!test-softlayer-ubuntu1604-x64-1"
   tasks:
     - name: remove node and npm packages
-      when: not os|startswith("win") and not os|startswith("zos")
+      when: not os|startswith("win") and not os|startswith("zos") and not os|startswith("ibmi") and os != "centos5"
```

Comment:
Why is there centos5 changes in this PR?

Reply:
I think it's not showing because this is still in draft but this PR needs to be rebased onto the current master as it has merge conflicts when I tried to pull down the changes locally. The centos5 stuff was removed in https://github.com/nodejs/build/pull/1984.